README.md

```diff
@@ -29,11 +29,33 @@ bundle exec rake
 ```
 note that some integration tests may hit a live server, for which you may need to be on the Stanford VPN.
 
+## Building services
+For development we can use Foreman to run a procfile, but on a deployed machine, we export the rules to systemd:
+```
+# disable/remove old rules
+sudo systemctl stop traject.target
+sudo systemctl disable traject.target
+
+# Ensure the .env file exists with JRUBY_OPTS=-J-Xmx8192m LANG=en_US.UTF-8 and then:
+foreman export -a traject -u indexer -f Procfile.stage --formation marc_bodoni_dev_indexer=1,marc_morison_dev_indexer=1,folio_dev_indexer=8,sw_dev_indexer=2,sw_preview_stage_indexer=2,earthworks_stage_indexer=1 systemd ~/service_templates
+sudo cp /opt/app/indexer/service_templates/* /usr/lib/systemd/system/
+
+sudo systemctl enable traject.target
+sudo systemctl start traject.target
+```
+
+## Monitor logs
+```
+ksu
+journalctl -u traject-marc_bodoni_dev_indexer.1.service -e
+```
+
 ## indexing data
 indexing is a multi-step process:
 1. an extractor process publishes data to be indexed to a [kafka](https://kafka.apache.org/) topic
-2. a daemon run by [eye](https://github.com/kostya/eye) consumes data from the kafka topic and invokes traject
+2. Systemd runs the various traject services
```

Comment:
don't the services still do the same thing? i think it's useful to note that they are consuming data from kafka to send to traject

Reply:
It its traject itself that is acting as a kafka consumer.    But the traject service it can have any input  you configure it to have.  Systemd knows nothing about kafka, so I didn't think that belongs here.